run.sh

```diff
@@ -1,13 +1,42 @@
 #!/bin/bash
```

Comment:
If this is getting updated can you add a small usage comment at the top of the script to say what the args are for?

Reply:
Also would need to be updated in the docs at https://phac-nml.github.io/irida-documentation/developer/getting-started/#running-and-building-irida